src/raw/iter.rs

```diff
@@ -145,6 +145,10 @@ impl<'a> TryInto<Bson> for RawElement<'a> {
 }
 
 impl<'a> RawElement<'a> {
+    pub fn is_empty(&self) -> bool {
```

Comment:
Thanks for catching this clippy error! I think it would make more sense to call the `len` method `size` since it's exposing the number of bytes in the element. That would remove the need to have this `is_empty` method, which likely won't have any utility for the users of this type.

Reply:
Oh that's a great point, in that case I think we'll want to stick to `len` for the name and change it with the next major release (RUST-1869). I still don't think it makes that much sense to have an `is_empty` method, so I suggest silencing the clippy error with `#[allow(clippy::len_without_is_empty)]`. Sorry for all of the back-and-forth here!

Reply:
done!